packages/zip-it-and-ship-it/package.json

```diff
@@ -44,7 +44,7 @@
     "@babel/parser": "^7.22.5",
     "@babel/types": "7.27.1",
     "@netlify/binary-info": "^1.0.0",
-    "@netlify/serverless-functions-api": "^1.41.2",
+    "@netlify/serverless-functions-api": "2.0.2",
```

Comment:
Are we sure about all the `package-lock.json` changes that come with this one? With just 1 upgraded package I wouldn't expect to see such a large diff in there, but also don't have all the context on what's being changed, so simply posing the question.

Reply:
yes, this brings in a major bump to all the otel packages as well 👍

Reply:
Happy to wait until you are back from pto to merge this, there is no rush for it to happen today

Reply:
All good from my end!

Reply:
Yeah, that's worth looking into.